packages/cli-template-contracts-hardhat/package.json

```diff
@@ -32,7 +32,7 @@
         "test:report-gas": "REPORT_GAS=true hardhat test",
         "test:coverage": "hardhat coverage",
         "typechain": "hardhat typechain",
-        "prepublish": "tar -czf files.tgz .gitignore",
+        "prepublish": "tar --exclude typechain-types --exclude artifacts --exclude cache  --exclude node_modules -czf files.tgz .gitignore .yarn .yarnrc.yml",
```

Comment:
I'm not sure if we need to explicitly exclude those folders. I've just tried to run `tar -czf files.tgz .gitignore` and even if there are those folders (typechain-types, artifacts, node_modules), they've not been included.

Reply:
If you run `semaphore create` and select the `monorepo-ethers` template for example, you will see the `artifacts` and `typechain-types` there inside `apps/contracts`. Then we can include those files to make sure they will never be there or make sure that the files are not there when releasing a new version of the packages. 

I think that with automatic releases, we wouldn't need to exclude the files.

Reply:
Sounds right for the monorepos, but we shouldn't need to exclude them in the contracts-hardhat template right?

Reply:
Yes, you're right. That's not necessary for this specific template, just for the others. I will update it.